### PR TITLE
chore: cleanup issue and PR templates

### PR DESCRIPTION
**Pull request by amrbashir**

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,30 +1,21 @@
...
+    - update docs
+    - fix bugs
 
-### Checklist
-- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
-- [ ] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md).
-- [ ] I have added a convincing reason for adding this feature, if necessary
-
-### Other information
+2. If there is a related issue, referenced in the PR text, e.g. closes #123.
```
Comment (FabianLars) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
2. If there is a related issue, reference it in the PR text, e.g. closes #123.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,30 +1,21 @@
...
+3. If this change requires a new version, then add a change file in `.changes` directory with the appropriate bump, see https://github.com/tauri-apps/tauri/blob/dev/.changes/readme.md
+4. Ensure that all your commits are signed https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits
+5. Ensure `cargo test` and `cargo clippy` passes.
+6. Open as a draft PR if your work is still in progress.
```
Comment (FabianLars) [inline .github/PULL_REQUEST_TEMPLATE.md]: Not quite sure about this one, i feel like there's missing a word idk.
Maybe `"Choose a draft PR if your work is still in progress."` or `"Propose your changes as a draft PR if your work is still in progress."`

**Review by FabianLars — COMMENTED** [review]
Yesss, finally i won't be the only one who deletes the whole template from their prs 😂

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (lucasfernog) [reply .github/PULL_REQUEST_TEMPLATE.md]: "Propose your changes as a draft PR if your work is still in progress." thumbs up

**Review by lucasfernog — APPROVED** [review]

**Pull request merged**